### PR TITLE
Feature/object attr

### PR DESCRIPTION
**Pull request by EliChin**
Add text from Confluence + about duplicated attributes.

### CODE REVIEW

File: 01-arch/03-objects.md [hunk 1]
```diff
@@ -1 +1,9 @@
...
+
+The maximum size for an object is fixed and can be changed only for the whole network in the main contract. It means that if a file is too heavy, it will be automatically divided into smaller objects (all of them containing the header of the initial object). This smaller parts are put in a container and placed to a Storage Node. Later, they can be assembled to the initial object. Such assembling is performed in the storage nodes upon a corresponding request for a linking :question_mark: object. Once your file is converted into an object (or several objects), this object cannot be changed.
+
+One can define the format of the object in an API Specification. For more information, see API Specification [link].
```
Comment (anatoly-bogatyrev) [inline 01-arch/03-objects.md:9]: Could you please add a link to the [link] or remove it?

**Review by anatoly-bogatyrev — CHANGES_REQUESTED** [review]

File: 01-arch/03-objects.md [hunk 2]
```diff
@@ -1 +1,9 @@
...
+ObjectID is a hash that equals Headers hashes plus Payload hashes. Any object includes a system header, extended headers, and a payload. A system header is an obligatory field, while extended headers may be omitted. However, any extended header should follow a particular structure (e.g. IntegrityHeader is a must). A user can add any extended header in the form of a key-value pair, though keeping in mind that it cannot be duplicated with several values. One attribute -- one value. Please note that any object initially has FileName, so that you cannot create an extended header with it as a key.
+
+The maximum size for an object is fixed and can be changed only for the whole network in the main contract. It means that if a file is too heavy, it will be automatically divided into smaller objects (all of them containing the header of the initial object). This smaller parts are put in a container and placed to a Storage Node. Later, they can be assembled to the initial object. Such assembling is performed in the storage nodes upon a corresponding request for a linking :question_mark: object. Once your file is converted into an object (or several objects), this object cannot be changed.
```
Comment (anatoly-bogatyrev) [inline 01-arch/03-objects.md:7]: Incorrect thesis - could you please remove (all of them containing the header of the initial object)?
Thanks

Comment (anatoly-bogatyrev) [inline 01-arch/03-objects.md:7]: Could you please remove :question_mark:? Thanks

**Review by anatoly-bogatyrev — CHANGES_REQUESTED** [review]

File: 01-arch/03-objects.md (hunk 2 above)
Comment (EliChin) [reply 01-arch/03-objects.md]: Done

Comment (EliChin) [reply 01-arch/03-objects.md]: Done

**Review by anatoly-bogatyrev — APPROVED** [review]

**Pull request merged**